deploy/charts/emqx/templates/secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "emqx.fullname" . }}-basic-auth
+  namespace: {{ .Release.Namespace }}
+type: kubernetes.io/basic-auth
+stringData:
+  username: admin
+  password: public
```

Comment:
Maybe we should configure it in `Value.yaml·, as far as I know, users can set their username and password by modifying emqx's environment variables

Reply:
prometheus operator needs both values to be present in a secret. (not a configmap) [AFAIK]